rpc/src/alpenglow_last_voted.rs

```diff
@@ -0,0 +1,64 @@
+// This is a temporay work around to correctly report last votes of peers in Alpenglow
+// before we have the certs in banks.
+use {
+    solana_clock::Slot,
+    solana_pubkey::Pubkey,
+    std::{collections::HashMap, sync::RwLock},
+};
+
+// We should only have 50 nodes in test cluster. By the time we move to testnet we
+// will have the real thing implemented.
+const MAX_ENTRIES: usize = 2000;
+
+#[derive(Default)]
+pub struct AlpenglowLastVoted {
+    last_voted_map: RwLock<HashMap<Pubkey, Slot>>,
+}
+
+impl AlpenglowLastVoted {
+    pub fn update_last_voted(&self, verified_votes_by_pubkey: &HashMap<Pubkey, Vec<Slot>>) {
```

Comment:
it seems we are only using the largest slot so I wonder if we can compute that instead of sending a list of slots and having to iterate over it again and again.

Reply:
Sure, let me see how Ashwin wants to change verified votes. But let me submit this and push to invalidator first to silence the annoying alerts.